packages/calypso-build/package.json

```diff
@@ -44,7 +44,7 @@
 		"@babel/preset-env": "7.7.6",
 		"@babel/preset-react": "7.7.4",
 		"@babel/preset-typescript": "7.7.4",
-		"@types/webpack-env": "1.14.1",
+		"@types/webpack-env": "1.15.0",
```

Comment:
Odd to see `types` as a dependency, but I think it's correct in this case because we're sharing this build tool that expects `webpack-env` types to be present.